src/views/splash/presentation.jsx

```diff
@@ -437,35 +418,23 @@ class SplashPresentation extends React.Component { // eslint-disable-line react/
                 ] : []}
                 {
                     this.props.sessionStatus === sessionActions.Status.FETCHED &&
-                    Object.keys(this.props.user).length === 0 && // if user is not logged in
-                    (ShowTopBanner ? [
+                    Object.keys(this.props.user).length === 0 && (// Only show top banner if user is not logged in
+                    /* eslint-disable indent, react/jsx-indent, react/jsx-indent-props */
```

Comment:
fwiw, i was able to indent this without getting lint errors

Reply:
```
                {
                    this.props.sessionStatus === sessionActions.Status.FETCHED &&
                    Object.keys(this.props.user).length === 0 && (// Only show top banner if user is not logged in
                        (Date.now() >= HOC_START_TIME && Date.now() < HOC_END_TIME) ? (
                            <MediaQuery
                                key="frameless-tablet"
                                minWidth={frameless.tabletPortrait}
                            >
                                <TopBanner />
                            </MediaQuery>
                        ) : (
                            <Intro
                                key="intro"
                                messages={messages}
                            />
                        )
                    )
                }
```

Reply:
Thanks!